### ISSUE TITLE
CAP 11 - ART 73 - PAR 7 

### ISSUE THREAD

**Issue by gabrieljm00** [CO-Intercomp/estatuto-intercomp#25]
tirar isso: com colete vira varzea

**Comment by LucasEPO** [comment]
Discordo, não acho q utilizar colete ou não faca diferenca, so acrescente uma dificuldade a mais, ainda mais pelo fato que a equipe não mandante deve apresentar outro uniforme em caso de muita semelhanca entre os uniformes.
Ai podemos somar ai que pelo formato do sorteio creio q tem equipes q tem mais jogos por ter iniciado nas quartas, e geralmente eles estao como não mandantes, tendo q assim aumentar a quantidade de uniformes que eles devem arrumar/levar enfim.
Acredito que a utilização de colete caso necessário não tornara varzea, mas sim ajudaria a manter a continuidade do campeonato.
